tests/zfs-tests/tests/functional/cli_root/zpool_create/cleanup.ksh

```diff
@@ -32,6 +32,12 @@
 . $STF_SUITE/include/libtest.shlib
 . $STF_SUITE/tests/functional/cli_root/zpool_create/zpool_create.shlib
 
-cleanup_devices $DISKS
+log_pass $DISKS
```

Comment:
This seems a little odd to have `log_pass` twice in here.

Reply:
Yes it is odd. This is a leftover from debugging. Will remove. By the way, this change was only needed to avoid the cleanup from failing due to:
```
Test: /home/atf/src/git/zfs-fix-zpool-create-dryrun/tests/zfs-tests/tests/functional/cli_root/zpool_create/cleanup (run as root) [00:00] [FAIL]
07:08:18.88 ERROR: parted /dev/var/tmp/file-vdev0 -s -- mklabel gpt exited 1
07:08:18.88 Error: Could not stat device /dev/var/tmp/file-vdev0 - No such file or directory.
```
So if you know of a better way to avoid this failure I'm all ears.

Reply:
Hmm, `cleanup.ksh` is still failing, looking into this right know.

Reply:
D'oh, now I realize that the problem is that `zero_partitions` is (incorrectly?) prepending `/dev` to the disk path and then fails to run parted on the non-existing file. The diskfiles itself do exist.
https://github.com/openzfs/zfs/blob/8f158ae6ad98d7b4f4c583a77a44009580dee91c/tests/zfs-tests/include/libtest.shlib#L900
Any idea how I can fix this? Unfortunately I've run out of ideas right now.

Reply:
Found the problem, I was running `zfs-test.sh -f` which disables block device tests, so I guess that part wasn't initialized.  Sorry for the noise.